### PR TITLE
Update page in Connect API Manager “impact-of-unpublishing-an-api”

### PR DESCRIPTION
**Pull request by sbolosan**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by sbolosan** [comment]
@jcollins-axway @kflan-io @dfeldick @vivekschauhan please review.  
Changes don't leave much meat on this bone.  The change of unpublishing really only is a status thing now.

**Comment by sbolosan** [comment]
Documentation story number - https://jira.axway.com/browse/APIGOV-16120

File: content/en/docs/central/connect-api-manager/impact-of-unpublishing-an-api.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ linkTitle: Unpublish an API
...
 description: Understand the implications of unpublishing your API in API Manager
-  in the centralMode (publishToEnvironmentAndCatalog), especially as it relates to subscriptions.
+  in the centralMode (publishToEnvironmentAndCatalog), especially as it relates
+  to subscriptions.
 ---
-
 ## Unpublishing your API in publishToEnvironmentAndCatalog mode
 
-Assume you have an API in API Manager that has been previously discovered by the agent and pushed to AMPLIFY Central as a Catalog item. Now you choose to unpublish that API in API Manager. If the agent's mode is set to `publishToEnvironmentAndCatalog`, then:
+Assume you have an API in API Manager that has previously been discovered by the agent and pushed to AMPLIFY Central as a Catalog item.  Now you choose to unpublish that API in API Manager. If the agent's mode is set to `publishToEnvironmentAndCatalog`, then:
```
Comment (dfeldick) [inline content/en/docs/central/connect-api-manager/impact-of-unpublishing-an-api.md:12]: This is really picky. But, I don't think that 2 spaces after a '.' are the norm anymore. That went out with typewriters. I just googled it.

Comment (sbolosan) [reply content/en/docs/central/connect-api-manager/impact-of-unpublishing-an-api.md]: Updated to remove space.

**Comment by lbadenhop** [comment]
@andreamussap Please bypass the cla check and merge the changes. Thanks

**Review by cbordier — APPROVED** [review]
I approve the changes
@lbadenhop, please proceed

**Pull request merged**